obspy/clients/fdsn/tests/test_mass_downloader.py

```diff
@@ -303,7 +303,8 @@ def test_channel_priority_filtering(self):
         c2 = Channel("10", "SHE", time_intervals)
         c3 = Channel("00", "BHZ", time_intervals)
         c4 = Channel("", "HHE", time_intervals)
-        channels = [c1, c2, c3, c4]
+        c5 = Channel("", "ELZ", time_intervals)
+        channels = [c1, c2, c3, c4, c5]
```

Comment:
Yes, yes I did - I have only added a small test for those low-gain cases though now. It will show up later when I push to this.